lib/PhantomEnvironment.js

```diff
@@ -457,7 +496,8 @@ PhantomEnvironment.prototype = _.create(Environment.prototype, /**@lends Phantom
             var args = [].slice.call(arguments);
             args = args.slice(3);
             this._page.onResourceRequested(callback, /* @covignore */ function (requestData) {
-                debug('Resource requested %s', requestData.url);
+                // todo: decide, remove or leave
+                // debug('Resource requested %s', requestData.url);
```

Comment:
Yep. We can cut this. That was made only for testing work which is related to loading resources.